azure/terraform/master-compliance-test.json

```diff
@@ -815,6 +902,180 @@
+                            "remediationFunction": "PR_AZR_0069_TRF.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "Azure storage account blob services diagnostic logs should be enabled",
+                    "description": "Storage Logging records details of requests (read, write, and delete operations) against your Azure blobs. The logs include additional information such as:_x000D_- Timing and server latency._x000D_- Success or failure, and HTTP status code._x000D_- Authentication details_x000D__x000D_This policy identifies Azure storage accounts that do not have logging enabled for blobs. As a best practice, enable logging for read, write, and delete request types on blobs.",
+                    "tags": [
+                        {
+                            "cloud": "git",
+                            "compliance": [],
+                            "service": [
+                                "terraform"
+                            ]
+                        }
+                    ]
+                },
+                {
+                    "masterTestId": "TEST_DIAGONASTIC_SETTINGS_5",
+                    "masterSnapshotId": [
+                        "TRF_TEMPLATE_SNAPSHOT"
+                    ],
+                    "type": "rego",
+                    "rule": "file(diagnosticsettings.rego)",
+                    "evals": [
+                        {
+                            "id": "PR-AZR-0070-TRF",
+                            "eval": "data.rule.log_queue",
+                            "message": "data.rule.log_queue_err",
+                            "remediationDescription": "In 'azurerm_monitor_diagnostic_setting' resource, set 'enabled = true' and category = 'auditevent' under 'log' block to fix the issue. please visit https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/monitor_diagnostic_setting#log for details.",
+                            "remediationFunction": "PR_AZR_0070_TRF.py"
+                        }
+                    ],
+                    "severity": "Medium",
+                    "title": "Azure storage account queue services diagnostic logs should be enabled",
+                    "description": "Queue logging cannot be enabled for Storage Accounts with 'kind' of BlobStorage **_x000D__x000D_Storage Logging records details of requests (read, write, and delete operations) against your Azure queues. The logs include additional information such as:_x000D_- Timing and server latency._x000D_- Success or failure, and HTTP status code._x000D_- Authentication details_x000D__x000D_This policy identifies Azure storage accounts that do not have logging enabled for queues. As a best practice, enable logging for read, write, and delete request types on queues.",
```

Comment:
you can use HTML code in the description blok

Reply:
@farchide ok if needed will use the html markup.